pmd-java/src/main/ant/alljavacc.xml

```diff
@@ -81,12 +82,42 @@ public class]]></replacevalue>
   }
 ]]></replacevalue>
         </replace>
-    	<replace file="${target}/net/sourceforge/pmd/lang/java/ast/Token.java">
-    		<replacetoken>public class Token</replacetoken>
-    	    <replacevalue><![CDATA[import net.sourceforge.pmd.lang.ast.GenericToken;
 
-public class Token extends GenericToken]]></replacevalue>
-    	</replace>
+        <replace file="${target}/net/sourceforge/pmd/lang/java/ast/Token.java">
+            <replacetoken>public class Token implements java.io.Serializable</replacetoken>
+            <replacevalue><![CDATA[import net.sourceforge.pmd.lang.ast.GenericToken;
+import net.sourceforge.pmd.lang.ast.RegionByLine;
+import net.sourceforge.pmd.lang.ast.RegionByLineImpl;
+
+public class Token implements GenericToken, java.io.Serializable]]></replacevalue>
+        </replace>
+
+        <!--Add implementation methods of GenericToken-->
+        <replace file="${target}/net/sourceforge/pmd/lang/java/ast/Token.java">
+            <replacetoken>specialToken;</replacetoken>
```

Comment:
this seems a little too generic... remember this will replace all occurrences of `specialToken;` with your contents